OpenRA.Game/Graphics/SpriteCache.cs

```diff
@@ -112,6 +112,9 @@ public void LoadReservations(ModData modData)
 				}
 			}
 
+			spriteReservations = new();
+			reservationsByFilename = new();
```

Comment:
perhaps add a comment explaining why we are creating new dictionaries

Reply:
Switched to a Clear+TrimExcess combo which should be equally descriptive as a comment